extension/spark-doris-connector/src/main/java/org/apache/doris/spark/cfg/ConfigurationOptions.java

```diff
@@ -0,0 +1,54 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.spark.cfg;
+
+public interface ConfigurationOptions {
+    // doris fe node address
+    String DORIS_FENODES = "doris.fenodes";
+
+    String DORIS_DEFAULT_CLUSTER = "default_cluster";
+
+    String TABLE_IDENTIFIER = "table.identifier";
+    String DORIS_TABLE_IDENTIFIER = "doris.table.identifier";
+    String DORIS_READ_FIELD = "doris.read.field";
+    String DORIS_FILTER_QUERY = "doris.filter.query";
+    String DORIS_FILTER_QUERY_IN_VALUE_MAX = "doris.filter.query.in.value.max";
```

Comment:
This configuration item must less than 10000, can you impose restrictions on this item

Reply:
done